apix-impl/src/main/java/eu/xenit/apix/alfresco/search/SearchFacetsServiceImpl.java

```diff
@@ -60,12 +64,35 @@
     // Intellij can't handle this file being reused in different libs.
     @Autowired
     public SearchFacetsServiceImpl(ServiceRegistry serviceRegistry) {
```

Comment:
You can use `@Autowired` on a constructor with more parameters instead of autowiring the private fields and then adding some setters to set mocks in the test.